### ISSUE TITLE
Fix Relationships with BlazeFrontEnd Repository

### ISSUE THREAD

**Issue by Bcsotty** [Fire-Alarm-App/FireAlarmApp#5]
This issue references this message in the discord:
okay I was speaking with Andy and did some research, what we are currently doing on the server is apparently very bad for the PWA. We shouldn't be running the app using the react development server, as this is built for development and will run slower and could potentiall provide information to users that we don't want them to see. What we should do instead and I just tested this to make sure it works, is just run npm run build on the react server to let it build to a static directory, then we include this static directory in the public folder of the FireAlarmApp. I created a PR with instructions and this change implemented, but we do need to make some changes to the BlazeFrontEnd app to help with readability. Whenever you are feeling better @LuffyOS | PHD '29, can you add any changes you made to get it to work on the school server into the BlazeFrontEnd repository? I will clean it up so that we no longer have this mess and this should help integration in the future. 

TL:DR
The BlazeFrontEnd repository should only host the latest ReactJS code and the FireAlarmApp will have the latest version of the BlazeFrontEnd repository saved as a static build instead of running the ReactJS development server with the Express server for security and performance reasons

I will be adding to the readme and integrating the react server for future development
